Menu/Menu.js

```diff
@@ -1,10 +1,10 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+  menu[0].classList.toggle("menu--open");
+};
+
+const menu = document.querySelectorAll(".menu");
```

Comment:
```suggestion
const menu = document.querySelector(".menu");
```
We know we only want the first thing with the .menu class, so I would just use querySelector rather than query selector all, then you don't have to use bracket notation to access the element you want.